zuul.d/tcib.yaml

```diff
@@ -0,0 +1,19 @@
++---
++- job:
++    name: cifmw-base-tcib
```

Comment:
Please call this job here: https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/projects.yaml#L9 so that we can see the results